app/dawn-hub/Views/OnboardingViews/Step2_BlockSettingsView/BlockSettingsView.jsx

```diff
@@ -230,7 +225,7 @@ class BlockSettingsView extends Component {
 						<div className="BlockSettingsView_checkboxBlock">
 							<ToggleCheckbox
 								className="BlockSettingsView_checkbox"
-								checked={recommendedChoices}
+								checked={this.recommendedChoicesActive()}
 								onChange={() => this.toggleRecommendedChoices(!recommendedChoices)}
 							/>
 							<div className="BlockSettingsView_checkboxLabel" onClick={() => this.toggleRecommendedChoices(!recommendedChoices)}>{t('ghostery_dawn_onboarding_recommended_choices')}</div>
```

Comment:
(Sorry if this wasn't initially clear) - let's follow through on this idea throughout the component and get rid of `recommendedChoices` completely. We can call `this.recommendedChoicesActive()` one time at the beginning of `render` and save the result to avoid repeating the calculation three times

Reply:
Ah ok, yea that makes sense!